libutils/logging.c

```diff
@@ -174,6 +179,11 @@ static void LogToConsole(const char *msg, LogLevel level, bool color)
     }
     else
     {
+        if (color)
+        {
+            fprintf(stdout, "%s", LogLevelToColor(level));
```

Comment:
General pattern: any use of `fprintf(tgt, "%s", blah)` is more naturally written `fputs(blah, tgt)` which skips parsing of the format string and uses faster simpler code paths.  (A good compiler might even make this transformation for you, but it's better to write it the efficient way to begin with.)